app/workers/subject_set_subject_counter_worker.rb

```diff
@@ -0,0 +1,19 @@
+class SubjectSetSubjectCounterWorker
+  include Sidekiq::Worker
+
+  sidekiq_options congestion: {
+    interval: 1.hour,
+    max_in_interval: 10,
+    min_delay: 3.minutes,
+    reject_with: :cancel,
```

Comment:
This job is called with a `perform_in(3.minutes, ...)` from the SetMemberSubject's after create callback, so it should only be invoked after the last SMS is created.